tox.ini

```diff
@@ -5,14 +5,16 @@ max-line-length = 100
 
 [tox]
 install_command = pip install {opts} {packages}
-envlist = lint,clean,py27-{1.7,1.8,1.9}-{postgres,sqlite},stats
+envlist = lint,clean,py27-{1.7,1.8,1.9}-{postgres,sqlite},py35-{1.8,1.9}-{postgres,sqlite},stats
 
 [testenv]
 usedevelop = True
 commands =
   {envbindir}/coverage run -p --omit="*tests*" --source=predicate --branch \
     setup.py test
-basepython = python2.7
```

Comment:
This seems to have broken the `lint` tox environment. I think you need to define a `basepython` parameter. How about updating this to:
```
basepython =
    python3.5
    py27: python2.7
```